ha/setup/v2/ha_setup.py

```diff
@@ -0,0 +1,215 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import pathlib
+import os
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.log import Log
+from cortx.utils.validator.v_pkg import PkgV
+from cortx.utils.security.cipher import Cipher, CipherInvalidToken
+sys.path.append(os.path.join(os.path.dirname(pathlib.Path(__file__)), '..', '..', '..'))
```

Comment:
Do I understand correctly that this is needed for `ha` package import?
If yes, maybe instead `ha` package can be installed and used as a usual package?
Though, we don't have setup.py here and I personally don't know whether pyinstaller provides such feature.

Reply:
@azheregelya 
>Do I understand correctly that this is needed for ha package import?
yes it is for ha package


Here pyinstaller is same as setup.py but it covert code to binary which was used previously when code was not open source. Currently we don't have setup.py. But due to this line we can also run python code from repo directly.

Reply:
I think we are good with removal of this line. Please let me know if there are any issues.